text/0000-windows-subsystem.md

```diff
@@ -0,0 +1,161 @@
+- Feature Name: Windows Subsystem
+- Start Date: 2016-07-03
+- RFC PR: ____
+- Rust Issue: ____
+
+# Summary
+[summary]: #summary
+
+Rust programs compiled for windows will always flash up a console window on
```

Comment:
Personally I would use the terminology `allocate a console window`, due to the behavior matching `AllocConsole`.